### ISSUE TITLE
SDK error on pycharm project creation

### ISSUE THREAD

**Issue by pswenson** [aws/aws-toolkit-jetbrains#1160]
**Describe the bug**
after project creation I got an error concerning SDK setup

**Expected behavior**
new project ready to go with configured SDK

**Screenshots**
![image](https://user-images.githubusercontent.com/430370/62984760-a8e72780-bdf1-11e9-8aee-0f9d748bd617.png)

**Your Environment**
 - OS: macos 
 - JetBrains' Product: pycharm
 - JetBrains' Product Version: 2019.2
 - Toolkit Version: 1.5
 - SAM CLI Version: 0.19.0
 - JVM/Python Version: python 3.7

**Comment by pswenson** [comment]
regarding error above, the python binary exists via symlink

```
:➜  bin ls -al
total 136
drwxr-xr-x  18 pswenson  1909375316    576 Aug 13 17:30 .
drwxr-xr-x   7 pswenson  1909375316    224 Aug 13 17:30 ..
-rw-r--r--   1 pswenson  1909375316   2141 Aug 13 17:30 activate
-rw-r--r--   1 pswenson  1909375316   1151 Aug 13 17:30 activate.csh
-rw-r--r--   1 pswenson  1909375316   3097 Aug 13 17:30 activate.fish
-rw-r--r--   1 pswenson  1909375316   1540 Aug 13 17:30 activate.ps1
-rw-r--r--   1 pswenson  1909375316   1029 Aug 13 17:30 activate.xsh
-rw-r--r--   1 pswenson  1909375316   1512 Aug 13 17:30 activate_this.py
-rwxr-xr-x   1 pswenson  1909375316    284 Aug 13 17:30 easy_install
-rwxr-xr-x   1 pswenson  1909375316    284 Aug 13 17:30 easy_install-3.7
-rwxr-xr-x   1 pswenson  1909375316    266 Aug 13 17:30 pip
-rwxr-xr-x   1 pswenson  1909375316    266 Aug 13 17:30 pip3
-rwxr-xr-x   1 pswenson  1909375316    266 Aug 13 17:30 pip3.7
lrwxr-xr-x   1 pswenson  1909375316      9 Aug 13 17:30 python -> python3.7
-rwxr-xr-x   1 pswenson  1909375316   2371 Aug 13 17:30 python-config
lrwxr-xr-x   1 pswenson  1909375316      9 Aug 13 17:30 python3 -> python3.7
-rwxr-xr-x   1 pswenson  1909375316  12552 Aug 13 17:30 python3.7
-rwxr-xr-x   1 pswenson  1909375316    262 Aug 13 17:30 wheel
:➜  bin pwd
/Users/pswenson/.local/share/virtualenvs/sam-1ywvqlN_/bin
```

**Comment by abrooksv** [comment]
Our 1.6-EAP.2019.08.14 contains the fix: https://plugins.jetbrains.com/plugin/11349-aws-toolkit/update/67195

Direct download / One time install: https://plugins.jetbrains.com/plugin/download?rel=true&updateId=67195
Opt-in instructions for EAP channel: https://github.com/aws/aws-toolkit-jetbrains#eap-builds

**Comment by pswenson** [comment]
sorry I got different errors when I tried again (also logged).  If I see this again, I'll add in the log.

**Comment by zhangzhx** [comment]
Hi @pswenson , what is the different error you've seen? This issue should be fixed in the latest [EAP version](https://plugins.jetbrains.com/plugin/11349-aws-toolkit/update/67554). Can you give it a try and let us know if you still see this issue? Thanks.

**Comment by pswenson** [comment]
I gave it a try with the latest EAP and the error did not occur.  Thanks!

There was a different issue, probably unrelated where when I picked the Hello World template, the template files weren't created.

**Comment by abrooksv** [comment]
How long did you wait? I don't think they generate till after the python sdk is indexed.

**Comment by pswenson** [comment]
tried again and saw no files.... and no sign of indexing after 10 secs or
so.  I waited for a full 15 mins just to make sure

**Comment by abrooksv** [comment]
We have fixed these issues in 1.6, please re-open if still facing issues after upgrading
